README.md

```diff
@@ -1,9 +1,11 @@
 # [emberclear](https://emberclear.io)
 
 emberclear is published at: https://emberclear.io
-and can be run locally with docker via
+and can be run locally via
 ```
-docker run -d -p 4201:80 nullvoxpopuli/emberclear
+cd emberclear/packages/frontend
```

Comment:
I think this should maybe go under a "Development" or "Contributing" section?

The first two lines here were/are intended for consumers/users/non-devs

Reply:
Good call, moved to CONTRIBUTING.md